document-generation/smartdocuments/src/main/kotlin/com/ritense/smartdocuments/client/SmartDocumentsClient.kt

```diff
@@ -50,6 +69,38 @@ class SmartDocumentsClient(
         }
     }
 
+    fun generateDocumentStream(
+        smartDocumentsRequest: SmartDocumentsRequest,
+        outputFormat: DocumentFormatOption,
+    ): FileStreamResponse {
+        try {
+            val bodyFlux = webClient().post()
+                .uri("/wsxmldeposit/deposit/unattended")
+                .contentType(APPLICATION_JSON)
+                .bodyValue(smartDocumentsRequest)
+                .retrieve()
+                .bodyToFlux<DataBuffer>()
+
+            val tempFile = Files.createTempFile("smartDocumentsResponse", ".tmp")
```

Comment:
I believe the story mentioned a 'Temporary file storage solution needs to be created as a separate "feature" built in core(?)'. I don't think this is what was intended.

Reply:
done